rbac/migration_tool/migrate.py

```diff
@@ -152,6 +154,19 @@ def migrate_data_for_tenant(tenant: Tenant, exclude_apps: list, replicator: Rela
         logger.info(f"Migration completed for role: {role.name} with UUID {role.uuid}.")
     logger.info(f"Migrated {roles.count()} roles for tenant: {tenant.org_id}")
 
+    public_default_roles = Role.objects.filter(platform_default=True, tenant=Tenant.objects.get(tenant_name="public"))
+
+    with transaction.atomic():
+        for group in tenant.group_set.all():
```

Comment:
I wonder if we can combine this with migrate_users_for_groups in some way, which already iterates through (almost) all tenant groups.